base/compiler/typelattice.jl

```diff
@@ -28,11 +28,11 @@ end
 # end
 # ```
 mutable struct Conditional
-    var::Union{Slot,SSAValue}
+    var::Slot
```

Comment:
Why can't this apply to SSAValues?

Reply:
Not worth the compile-time performance cost, since it'll never generate better results (on our current syntax).